Automattic/vip-cli
# Remove check on importing sql to k8s sites

## Description
Removes check that prevents sql imports on k8s residents

## Steps to Test

1. Check out PR.
1. Run `npm run build`
1. Run `./dist/bin/vip-import-sql.js @<id> file.sql` where id is a k8s-based site 
1. Verify import is not prevented from running.
